.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 site/accounts/*.php
+.DS_Store
```

Comment:
This goes in your global `.gitignore`: https://davidwalsh.name/global-gitignore

Otherwise we’d have to add all temporary files used by any text editor the developers use, which becomes unwieldy. Using a global `.gitignore`, each developer makes sure their own development environment does not accidentally bleed into Git.